src/components/NavBar.vue

```diff
@@ -1,20 +1,68 @@
 <template>
-  <header class="bg-gray-200 py-6">
-    <div class="container m-auto flex flex-wrap jutify-end grid grid-cols-3">
+  <header class="pt-8 pb-12">
+    <nav 
+      :class="{ 'scrolled': !view.atTopOfPage }"
+      class="fixed flex w-full bg-white border-b items-center justify-between flex-wrap p-8 m-auto top-0 animated"
+    >
         <div>
-          <router-link class="link-gray px-2 uppercase tracking-widest" to="/">Good Bread</router-link>
+          <router-link
+            v-on:click.native='selectPage(1)'
```

Comment:
Updated the nav bar to be fixed when scrolling and to change color to reflect the page that the user is on